polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -201,53 +206,31 @@ parameter_types! {
 }
 
 impl frame_system::Config for Runtime {
-	/// The identifier used to distinguish between accounts.
```

Comment:
Removed the bad documentation and re-order the definitions to be the same as the upstream statemine runtime.